_data/simple-icons.json

```
@@ -4162,6 +4162,11 @@
             "hex": "E64415",
             "source": "https://www.spiegel.de"
         },
+        {
+            "title": "Detekt",
```

```suggestion
            "title": "detekt",
```

This is how they capitalize it.